doc/manual/manual.tex

```diff
@@ -1756,6 +1756,127 @@ \subsection{Compiling a static \aspect{} executable}
+\end{verbatim}
+
+\item Download and install the parallel \dealii{} binary
+%Current latest deal.ii version is 8.2.1.
+{\footnotesize
+\begin{verbatim}
+%  open \
+https://github.com/dealii/dealii/releases/download/v8.2.1/dealii-8.2.1-parallel-bundle.dmg
+\end{verbatim}
+}
+Open the downloaded disk image, and drag \dealii{}.app into the Applications folder.
+ To start the \dealii{} app, double click the icon in the Applications folder or use the open command:
+ \begin{verbatim}
+%   open /Applications/deal.II.app
+\end{verbatim}
+\dealii{} app opens a terminal window and displays a \dealii{} message.   \dealii{}.app will install all required libraries for \aspect{} (p4est, parMeTiS, and Trilinos) and will include the environment variables needed to use these libraries.
+
+\item Download  the  \aspect{} source from the git repository.
+
+Note: you {\bf MUST} \underline{build} and \underline{run} \aspect{} in the terminal window started by \dealii{}.
+
+\item Build \aspect{}
+
+\begin{enumerate}
+\item Go to the  directory where you wish to install \aspect{}  and run the following commands:
+
+{\footnotesize
+\begin{verbatim}
+> ccmake .
+press 'c'   
+ \end{verbatim}}
+
+This should display something like:
+
+{\footnotesize
+\begin{verbatim}
+Project  aspect  set up with  deal.II-8.2.1  found at /Applications/deal.II.app/Contents/Resources
+\end{verbatim}}
+
+exit cmake, then type 
+
+{\footnotesize
+\begin{verbatim}
+> cmake .
+ \end{verbatim}}
+
+This should display something like:
+
+{\footnotesize
+\begin{verbatim}
+====================================================
+============ Configuring ASPECT ====================
+====================================================
+-- Found deal.II version 8.2.1 at '/Applications/deal.II.app/Contents/Resources/lib/cmake/deal.II'
+-- Using PETSc = 'OFF'
+-- Added 184 tests
+====================================================
+...
+\end{verbatim}}
+
+\item Run make:
+
+{\footnotesize
+\begin{verbatim}
+bash-3.2$ make
+Scanning dependencies of target aspect
+[  0%] Building CXX object CMakeFiles/aspect.dir/source/adiabatic_conditions/initial_profile.cc.o
+[  0%] Building CXX object CMakeFiles/aspect.dir/source/adiabatic_conditions/interface.cc.o
+...
+Linking CXX executable aspect
+[100%] Built target aspect
+bash-3.2$ ls -l aspect
+-rwxr-xr-x  1 matsui  staff  19131292 May  7 15:02 aspect
+\end{verbatim}
+}
+There may be  warnings from the compiler, but if the \aspect{} target is created then it was successful.
+
+\item By default,  \aspect{}  compiles the debug version of the code.
+To compile the optimized version:
+{\footnotesize\begin{verbatim}
+> make release 
+\end{verbatim}}
+
+\end {enumerate}
+
+
+\item Run \aspect{}.
+
+A reminder: you {\bf must} run  \aspect{}  on the terminal window which is opened by \dealii{}.app.
+
+To start \aspect{} using MPI for parallelization:
+{\footnotesize
+\begin{verbatim}
+mpirun -np <# of processes> $HOME/src/aspect/build_deal.ii.app/aspect <parameter file>
```

Comment:
How come ASPECT is inside a ```build_deal.ii.app``` directory? Does this happen automatically on MAC? (I doubt it)